ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-ospf.j2

```diff
@@ -12,6 +12,17 @@ router ospf {{ process_id }}
 {%     if router_ospf.process_ids[process_id].router_id is arista.avd.defined %}
    router-id {{ router_ospf.process_ids[process_id].router_id }}
 {%     endif %}
+{%     if router_ospf.process_ids[process_id].distance is arista.avd.defined %}
```

Comment:
This logic looks good.

But as of now this if check is doing nothing. So this if check can be removed as the standalone lines 16 - 24 should work with out the outer if check for ```.distance```

Reply:
The check is only to avoid processing 3 `if`s if this knob is not even defined.